jitsi-utils/src/main/java/org/jitsi/utils/stats/RateStatistics.java

```diff
@@ -102,45 +104,42 @@ private synchronized void eraseOld(long nowMs)
         oldestTime = newOldestTime;
     }
 
-    public long getRate()
+    public synchronized long getRate()
```

Comment:
We use this class in some critical-path areas, right?  We may want to do some testing before adding a bunch of `synchronized`.

Reply:
All of these methods call `eraseOld`, which is already synchronized, so there are no additional lock aquisitions. If we wanted to optimize to avoid aqcuiring a lock we know we already hold (the impact of which is minimal AFAIK) we could remove some of the `synchronized`, but:
1. We'd have a warning to supress and it makes the code easier to break
2. These are pretty simple cases that the compiler could catch and optimize automatically, so I suspect it does so (I haven't checked)